components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```diff
@@ -1528,7 +1525,7 @@ paths:
         - API Revisions
       summary: Retrieve Revision
       description: |
-        Retrieve a revision of an API
+        Retrieve a revision of an API (This resource is not supported at the moment)
```

Comment:
Any reason why this resource is not supported? backend error?

Reply:
This is not supported since we do not use this in the publisher portal.

Reply:
Even if we don't use it, there can be customers who are using it. Therefore I don't think it's good to mark them as not supported at the moment. In fact, we support the resource from the backend and the only thing is we don't use it from our current UI.

Reply:
We do not support this from backend as well.